scripts/build/Platform/Windows/build_config.json

```diff
@@ -239,7 +239,29 @@
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "TEST_SUITE_periodic",
       "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo",
-      "CTEST_OPTIONS": "-L \"(SUITE_periodic)\" -T Test --no-tests=error",
+      "CTEST_OPTIONS": "-L \"(SUITE_periodic)\" -LE \"(REQUIRES_gpu)\"  -T Test --no-tests=error",
+      "TEST_METRICS": "True",
+      "TEST_RESULTS": "True"
+    }
+  },
+  "periodic_test_gpu_profile": {
+    "TAGS": [
+        "nightly-incremental",
+        "nightly-clean",
+        "weekly-build-metrics"
+    ],
+    "PIPELINE_ENV":{
+      "NODE_LABEL":"windows-gpu"
+    },
+    "COMMAND": "build_test_windows.cmd",
+    "PARAMETERS": {
+      "CONFIGURATION": "profile",
+      "OUTPUT_DIRECTORY": "build\\windows",
+      "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0",
+      "CMAKE_LY_PROJECTS": "AutomatedTesting",
+      "CMAKE_TARGET": "TEST_SUITE_periodic",
+      "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo",
+      "CTEST_OPTIONS": "-L \"(SUITE_periodic_REQUIRES_gpu)\" -T Test --no-tests=error",
```

Comment:
`SUITE_periodic_REQUIRES_gpu` appears to be an invalid combination of two separate identifiers. Notably, CTest is only able to perform a union of labels, so I believe you'll want to use:
```
-L REQUIRES_gpu -LE "(SUITE_main | SUITE_smoke | SUITE_benchmark | SUITE_sandbox)"
```

This will need verification+iteration when the infrastructure for it is coming online, so I don't think this PR needs to be blocked.

Reply:
Ran a branch job for both of these periodic test configurations after submitting this change to build_config.json; they both behaved as expected.  Also, the main GPU job uses this same label combination for its inclusion regex:
https://github.com/o3de/o3de/blob/da421b7056eeeaeb1869d3c22c5acf5c6bcd3096/scripts/build/Platform/Windows/build_config.json#L172

Don't mind changing it if this is not the "ideal" filtering method but seems to work as is.

Reply:
I didn't know that was valid, but if it works then I'm satisfied :)